tests/Unit/Fixture/courses_section_response.json

```diff
@@ -0,0 +1,22 @@
+[
+    {
+        "SectionId": 501471,
+        "Name": "Section 1",
+        "Code": "sec1",
+        "Description": {
+            "Text": "",
+            "Html": ""
+        },
+        "Enrollments": []
```

Comment:
@jolezg it'd be great if you could find some example which has these enrollments